TOOLS/lua/autocrop.lua

```diff
@@ -214,19 +284,21 @@ function on_start()
     end
 
 
-    -- Verify if there is enough time for autocrop.
-    local time_spent = options.auto_delay + options.detect_seconds
-    local is_time_enough = time_spent < mp.get_property_native("playtime-remaining")
+    -- If it is the beginning, wait for detect_crop after auto_delay seconds, otherwise immediately.
+    local is_delay_needed = options.auto_delay > mp.get_property_native("playback-time")
+
+    if is_delay_needed then
 
-    if not is_time_enough then
-        mp.msg.warn("Not enough time to crop.")
-    end
+        -- Verify if there is enough time for autocrop.
+        local time_needed = options.auto_delay
```

Comment:
might as well add `detect_seconds` to avoid running the first timer only to find out there's not enough time later